### ISSUE TITLE
Clarification for SC 1.3.5 - Identify Input Purpose about control groups

### ISSUE THREAD

**Issue by giacomo-petri** [w3c/wcag#2818]
ACT rules discussion: https://github.com/act-rules/act-rules.github.io/issues/1967

Question: summarizing, is the intent of 1.3.5 to also check if the autocomplete value is allowed for the specific input type/control group or just to check if the autocomplete value reflects the input purpose?

Frequent scenarios of autocomplete attribute values that do not match their control group are:
* autocomplete="bday-day" with type="tel" (especially related to mobile experience)
* autocomplete="street-address" with type="text" 

Are they passing or failing 1.3.5?

**Comment by alastc** [comment]
Is [my comment](https://github.com/act-rules/act-rules.github.io/issues/1967#issuecomment-1332854491) a sufficient answer?

**Comment by giacomo-petri** [comment]
Thanks, closing
